.github/workflows/iroha2-dev-pr-title-commit-msg.yml

```diff
@@ -92,3 +92,70 @@ jobs:
       - name: None of the above
         if: steps.fix-match.outputs.match == '' && steps.refactor-match.outputs.match == '' && steps.feature-match.outputs.match == '' && steps.docs-match.outputs.match == '' && steps.ci-match.outputs.match == ''
         run: exit 1
+
+  commit-msg-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Feature
+        id: feature-match
+        continue-on-error: true
+        uses: gsactions/commit-message-checker@v1
+        with:
+          pattern: '^\[feature\] #\d+(, #\d+)*: .+$'
+          error: 'Wrong [feature] commits message'
+          excludeDescription: 'true'
+          excludeTitle: 'true'
+          checkAllCommitMessages: 'true'
+          accessToken: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
These options are mandatory if we want to check all commits associated with the current Pull Request and with `on: pull_request` Action trigger.